.changes/unreleased/tinymce-TINY-10590-2024-05-21.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Text content could move unexpectedly when deleting a paragraph.
```

Comment:
Should there be two changelog items?

Reply:
Probably not, seems to have tagged along with the cherrypick. Which one do you want removed?